### PR TITLE
[Suggestion] Configuration update

### PR DESCRIPTION
**Pull request by Mxrcon**
# [Suggestion] Configuration update

Hey @abhi18av, I took some time to update the configurations for this project, i also created profiles file to include on the main configuration. I put sample profiles for aws, azure and gcp that the users could modify. I removed the configuration from modules and put them on the standard profile.

## PR content
* Profiles (aws,azure.gcp,standard and labbactfiocruz)
* Taskfile

## Ending

Feel free to contact me if you have any doubt, critic or suggestion. I'm at your disposal to make any changes that you request.

Hope we have a mege!
Regards, Davi

### CODE REVIEW

File: Taskfile.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# https://taskfile.dev/
+
+version: '3'
+
+tasks:
+  default:
+    cmds:
+      - NXF_VER=21.04.0-EDGE nextflow run main.nf -params-file params/params.yml -resume -with-tower
```
Comment (abhi18av) [inline Taskfile.yml]: Let's update this to the latest version.

**Review by abhi18av — CHANGES_REQUESTED** [review]
Some minor changes before we can move forward with this PR.

File: Taskfile.yml (hunk 1 above)
Comment (Mxrcon) [reply Taskfile.yml]: ok! :+1:

**Review by abhi18av — APPROVED** [review]
Thanks for accomodating the change request Davi!

**Comment by Mxrcon** [comment]
@abhi18av Thank you too ! :tada:

**Pull request merged**